src/components/browse/components/SortController.js

```diff
@@ -300,21 +300,25 @@ const MultiColumnSortOption = React.memo(function MultiColumnSortOption(props){
                     }
                 </DropdownButton>
             </div>
-            <div className="col-3 pl-0">
-                <DropdownButton title={sortOrderTitle} variant={(variant ? variant + " " : "") + "btn-block text-left"}
+            <div className="col-3 ps-0">
+                <DropdownButton title={sortOrderTitle} variant={(variant ? variant + " " : "") + "w-100 text-start"}
                     size={size} onSelect={handleSortOrderSelection}>
                     <DropdownItem key="sort-direction-asc" eventKey={index + "|asc"}>Ascending</DropdownItem>
                     <DropdownItem key="sort-direction-desc" eventKey={index + "|desc"}>Descending</DropdownItem>
                 </DropdownButton>
             </div>
-            <div className="col-lg-1 col-2 pl-0">
+            <div className="col-lg-1 col-2 ps-0">
                 {!isLastRow ?
-                    <button type="button" className={`btn btn-${variant} btn-${size} btn-block`} onClick={onRemoveClick} data-tip="Remove sort column">
-                        <i className="icon icon-fw fas icon-minus w-100" />
-                    </button> :
-                    <button type="button" className={`btn btn-primary btn-${size} btn-block`} onClick={handleSettingsApply} data-tip="Re-sort columns">
-                        <i className="icon icon-fw fas icon-check w-100" />
-                    </button>}
+                    <div className="d-grid gap-1">
+                        <button type="button" className={`btn btn-${variant} btn-${size}`} onClick={onRemoveClick} data-tip="Remove sort column">
+                            <i className="icon icon-fw fas icon-minus w-100" />
+                        </button>
+                    </div> :
+                    <div className="d-grid gap-1">
+                        <button type="button" className={`btn btn-primary btn-${size}`} onClick={handleSettingsApply} data-tip="Re-sort columns">
+                            <i className="icon icon-fw fas icon-check w-100" />
+                        </button>
+                    </div>}
```

Comment:
FYI, according to migration [document](https://getbootstrap.com/docs/5.0/migration/#buttons) , this change looks required. Let me check it if I can simplify the usage.
> Breaking Dropped .btn-block for utilities. Instead of using .btn-block on the .btn, wrap your buttons with .d-grid and a .gap-* utility to space them as needed. Switch to responsive classes for even more control over them. [Read the docs for some examples.](https://getbootstrap.com/docs/5.0/components/buttons/#block-buttons)

Reply:
Re: auth0-lock

@Bianca-Morris we have been overriding SPC's auth0-lock using `overrides` section in package.json. For instance, fourfront is already [using](https://github.com/4dn-dcic/fourfront/blob/30a015d8b650ac98b217f0ee36469bf8780e5168/package.json#L150-L154) v12 where as smaht-portal and cgap-portal use legacy v11. Bumping SPC's auth0-lock to v12 makes the overrides unnecessary. Good catch, thanks!